src/store/currentUser.js

```diff
@@ -0,0 +1,13 @@
+
+
+export const CurrentUserModule = {
+    state: () => ({
+        currentUser: 0
```

Comment:
```suggestion
        currentUserIndex: 0
```
use meaningfull names. it's not current user, it's the index of current user. also this should be placed inside `allAccounts` module. there is no sense in dividing this logic between separate modules